pkg/planner/core/task.go

```diff
@@ -100,7 +100,7 @@ type copTask struct {
 }
 
 func (t *copTask) invalid() bool {
-	return t.tablePlan == nil && t.indexPlan == nil
+	return t.tablePlan == nil && t.indexPlan == nil && len(t.idxMergePartPlans) == 0
```

Comment:
BTW， what is the table plan means ?

Reply:
If I understand it correctly, table plan means subplan including TableRowIDScan or Selection/Aggregation on top of that TableRowIDScan.

Reply:
to avoid a case that doesn't have tablePlan and indexPlan, but indexMergePartPlans, current this case won't happen.

adding this is just make the guard line(won't affect any test case before)